### DISCUSSION TITLE
Question on the normalization method for calculating class activation maps.

### DISCUSSION THREAD

**Discussion by Minxiangliu** [Project-MONAI/MONAI#7468] [Q&A] [answered]
Thank you very much for providing a great graphical explanation tool, MONAI. I have a question regarding it. I'm currently using the [monai.visualize.CAM](https://docs.monai.io/en/stable/visualize.html#module-monai.visualize.class_activation_maps) tool to explain my model, which is [torchvision.models.efficientnet_b2](https://pytorch.org/vision/main/models/generated/torchvision.models.efficientnet_b2.html). Everything is running smoothly, but while understanding the code, I noticed in [monai.visualize.class_activation_maps.default_normalizer](https://docs.monai.io/en/stable/visualize.html#monai.visualize.class_activation_maps.default_normalizer) it mentions `This will flip magnitudes`. Could someone explain why? Why is this necessary?
Thanks in advance!

**Comment by KumoLiu** [comment, marked as answer]
Hi @Minxiangliu,
It's for the visualization, no special reason I think.
https://github.com/Project-MONAI/tutorials/blob/main/modules/interpretability/class_lung_lesion.ipynb
Thanks.
